lib/gettext_i18n_rails/model_attributes_finder.rb

```diff
@@ -63,7 +63,10 @@ def model_attributes(model, ignored_tables, ignored_cols)
     end
 
     def models
-      if Rails::VERSION::MAJOR > 2
+      if Rails::VERSION::MAJOR > 4
+        Rails.application.eager_load! # make sure that all models are loaded so that direct_descendants works
+        ApplicationRecord.direct_descendants
```

Comment:
this smells like trouble ... what if someone does not use ApplicationRecord or the models come from a gem ...
how about `models - ApplicationRecord` ?